nvme_gw.config

```diff
@@ -36,3 +36,5 @@ timeout = 60.0
 log_level = ERROR
 conn_retries = 3
 tgt_cmd_extra_args =
+transports = tcp
+transport_tcp_options =
```

Comment:
Instead of leaving it empty, I would turn this into a comment showing the syntax:

```suggestion
# transport_tcp_options = {"max_queue_depth" : 16, "max_io_size" : 4194304, "io_unit_size" : 1048576, "zcopy" : false}
```